package_managers/install_pkgs.bzl

```diff
@@ -97,7 +97,7 @@ docker load --input {base_image_tar}
 
 cid=$(docker run -d -v $(pwd)/{installables_tar}:/tmp/{installables_tar} -v $(pwd)/{installer_script}:/tmp/installer.sh --privileged {base_image_name} /tmp/installer.sh)
 
-docker attach $cid || true
+docker attach $cid
```

Comment:
This won't quite work either...

The || true was added because of a "race condition". Some of the containers exit so quickly, that "docker attach" returns an error because you can't attach to a stopped container.

We need to differentiate between errors from stopped containers that completed successfully and errors from the container itself.

Reply:
I see. I guess I'll just leave that there for now then.